lib/ansiblelint/utils.py

```diff
@@ -600,7 +622,7 @@ def append_skipped_rules(pyyaml_data, file_text, file_type):
         yaml_skip = _append_skipped_rules(pyyaml_data, file_text, file_type)
     except RuntimeError as exc:
         # Notify user of skip error, do not stop, do not change exit code
-        print('Error trying to append skipped rules: {!r}'.format(exc))
+        logger.error('Error trying to append skipped rules: {!r}'.format(exc))
```

Comment:
`exc` is not really needed
```suggestion
    except RuntimeError:
        # Notify user of skip error, do not stop, do not change exit code
        logger.exception('Error trying to append skipped rules')
```